src/Chainweb/Version.hs

```diff
@@ -897,7 +910,7 @@ skipFeatureFlagValidationGuard Mainnet01 h = h < 530500  -- ~ 2020-05-01T00:00:x
 skipFeatureFlagValidationGuard _ _ = False
 
 oldDaGuard :: ChainwebVersion -> BlockHeight -> Bool
-oldDaGuard Mainnet01 _ = True
-oldDaGuard Testnet04 _ = True
+oldDaGuard Mainnet01 h = h < 771_414 -- ~ 2020-07-23 16:00:00
+oldDaGuard Testnet04 h = h < 318_204 -- ~ 2020-07-23 16:00:00
```

Comment:
👍 double checked and got:

```
2020-07-23 16:02:52 UTC
2020-07-23 15:57:53 UTC
```

Reply:
Ahhh, I guess this is why the testnet fork was a week later.  👍